docs/whats-new/dotnet-7.md

```diff
@@ -0,0 +1,409 @@
+  - Note break for .NET 6 servicing
+  - XmlSecureResolver breaking change
+  - x86 host path on x64/arm64
+- [DataContractSerializer retains sign when deserializing -0](../core/compatibility/serialization/7.0/datacontractserializer-negative-sign.md)
+  - Note break for .NET 6 servicing
+  - DataContractSerializer breaking change
+- [Breaking changes in .NET 7](../core/compatibility/7.0.md)
+  - MemoryCache breaking change
+  - Networking breaking change
+  - Library support breaking change
+  - Socket.End* breaking change
+  - Remote auth provider breaking change
+  - Breaking change for OSArchitecture value
+  - WinForms obsoletions breaking change
+  - DataContractSerializer breaking change
+  - XmlSecureResolver breaking change
+  - Limit precision to 999,999,999
+  - Add ICU breaking change
+  - BinaryFormatter APIs obsolete as error
+  - Add trimming breaking change
+  - Default auth scheme handling
+  - X509ChainPolicy verification time
+  - Middleware deferral breaking change
+  - Kestrel HTTPS breaking change
+  - SignalR breaking change
+  - ContentRootPath breaking change
+  - x86 host path on x64/arm64
+  - ASP.NET Core 7 breaking change for MVC empty body detection
+  - ASP.NET Core 7 breaking change for MVC Logger Event IDs
+  - ASP.NET Core 7 breaking change for Microsoft.Data.SqlClient
+  - ASP.NET Core 7 breaking change for API Controller Actions DI
+  - ASP.NET Core 7 breaking change for SignalR Hub DI
+  - ASP.NET Core 7 breaking change for libuv
+- [Static abstract members declared in interfaces](../core/compatibility/core-libraries/6.0/static-abstract-interface-methods.md)
+  - Publish C# 11 speclets
+  - Generic math docs
+- [API obsoletions with non-default diagnostic IDs (.NET 7)](../core/compatibility/core-libraries/7.0/obsolete-apis-with-custom-diagnostics.md) - XmlSecureResolver breaking change
+- [Standard numeric format parsing precision](../core/compatibility/core-libraries/6.0/numeric-format-parsing-handles-higher-precision.md) - Limit precision to 999,999,999
+- [Globalization APIs use ICU libraries on Windows 10](../core/compatibility/globalization/5.0/icu-globalization-api.md) - Add ICU breaking change
+- [Changes to reflection invoke API exceptions](../core/compatibility/core-libraries/7.0/reflection-invoke-exceptions.md) - `ref` fields and `scoped`. Modify pages on struct creation, variable declaration, and ref struct.
+- [BinaryFormatter serialization methods are obsolete and prohibited in ASP.NET apps](../core/compatibility/core-libraries/5.0/binaryformatter-serialization-obsolete.md) - BinaryFormatter APIs obsolete as error
+- [Breaking changes in .NET Core 3.1](../core/compatibility/3.1.md) - x86 host path on x64/arm64
+
+## .NET Framework
+
+### Updated articles
+
+- [What's new in .NET Framework](../framework/whats-new/index.md) - `ref` fields and `scoped`. Modify pages on struct creation, variable declaration, and ref struct.
+
+## .NET fundamentals
```

Comment:
The .NET Fundamentals section is *really* long. 

We should sort by subfolder, and elide all the "breaking changes" issues.